_posts/entities/2020-12-07-env_surface_teleport.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: env_momentum_surface_teleport
+title: env__surface_teleport
```

Comment:
Doubled up underscores here
```suggestion
title: env_surface_teleport
```

Reply:
:+1:

Reply:
Should be fixup'd via interactive rebase instead of a new commit. Can be done in CLI with `git rebase -i origin/master`:
![image](https://user-images.githubusercontent.com/9014762/134793539-df28cbd2-fdd1-4065-b603-5b141a32b57b.png)
or more easily in a git client like Fork:
![image](https://user-images.githubusercontent.com/9014762/134793471-9cc5026d-c02e-432d-8494-6bcf4193806b.png)
Can also give it a rename if you want as well since commit msg also has double underscore, though not too important.

Reply:
i think it maybe worked?

Reply:
yup! nice :D

Reply:
poggers